### ISSUE TITLE
How to extract features from the last layer of Keras ResNet50 without ReLU layer 

### ISSUE THREAD

**Issue by Ahmadtrwh** [keras-team/keras#11702]
I am trying to extract the deep features using ResNet50 using this code:

from keras.applications.resnet50 import ResNet50
from keras.preprocessing import image
from keras.applications.vgg16 import preprocess_input
import numpy as np

model = ResNet50(weights='imagenet', include_top=True )
img_path = 'E:\project\KERAS DEEP\image.png'
img = image.load_img(img_path, target_size=(224, 224))
img_data = image.img_to_array(img)
img_data = np.expand_dims(img_data, axis=0)
img_data = preprocess_input(img_data)
model.summary()
model.layers.pop();
model.outputs = [model.layers[-1].output]
model.layers[-1].outbound_nodes = []
feature = model.predict(img_data)

the problem is that the variable **feature ** contains only the positive values becuse of ReLU layer, how i can extract the features withot the ReLU layer

**Comment by Skalextric** [comment]
Have you tried `model = ResNet50(weights='imagenet', include_top= **False** )`?

**Comment by see--** [comment]
```
from keras.models import Model
feature_model = Model(model.input, model.get_layer('add_61').output)
feature_model.predict(img_data)
```
